packages/functional-tests/tests/signin/signinBlocked.spec.ts

```diff
@@ -2,78 +2,68 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import {
-  test,
-  expect,
-  PASSWORD,
-  SIGNIN_EMAIL_PREFIX,
-  BLOCKED_EMAIL_PREFIX,
-} from '../../lib/fixtures/standard';
+import { test, expect } from '../../lib/fixtures/standard';
 
 test.describe('severity-2 #smoke', () => {
-  test.beforeEach(async () => {
-    test.slow(); //This test has steps for email rendering that runs slow on stage
-  });
-
   test.describe('signin blocked', () => {
-    test.use({
-      emailOptions: [{ prefix: BLOCKED_EMAIL_PREFIX, password: PASSWORD }],
+    test.beforeEach(async () => {
```

Comment:
I think we can lose the `async`
```suggestion
    test.beforeEach(() => {
```